src/main/java/com/ullim/ssomserver/domain/team/domain/Member.java

```diff
@@ -0,0 +1,47 @@
+package com.ullim.ssomserver.domain.team.domain;
+
+import com.ullim.ssomserver.domain.goal.domain.Goal;
+import com.ullim.ssomserver.domain.team.domain.type.TeamStatus;
+import com.ullim.ssomserver.domain.user.domain.User;
+import jakarta.persistence.*;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDate;
+
+@Entity
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Table(name = "tbl_member")
```

Comment:
관련도가 높은 순으로 배치하는 게 좋습니다.
```suggestion
@Getter
@NoArgsConstructor(access = AccessLevel.PROTECTED)
@Table(name = "tbl_member")\@Entity
@Entity
```